phys2bids/interfaces/txt.py

```diff
@@ -77,3 +73,121 @@ def populate_phys_input(filename, chtrig):
     timeseries.pop(0)
     ordered_timeseries = ordered_timeseries + timeseries
     return BlueprintInput(ordered_timeseries, freq, names, units)
+
+
+def acq_read(channel_list, chtrig, header=[]):
+    """
+    Reading function for acq files in txt format
+    Parameters
+    ----------
+    channel_list: list
+        list with channels only
+    chtrig : int
+        index of trigger channel
+    header: list
+        list with that contains file header
+    Returns
+    -------
+    BlueprintInput
+    See Also
+    --------
+    physio_obj.BlueprintInput
+    """
+    # get frequency
+    if len(header) == 0:
+        raise AttributeError('Files without header are not supported yet')
+    interval = header[1][0].split()
+    if interval[-1].split('/')[0] not in ['hr', 'min', 'sec', 'µsec', 'msec']:
+        raise AttributeError(f'Interval unit "{interval[-1]}" is not in a valid LabChart'
+                             'time unit, this probably means your file is not in labchart format')
+    interval[-1] = interval[-1].split('/')[0]
+    if interval[-1] != 'sec':
+        print('Interval is not in seconds. Converting its value.')
+        if interval[-1] == 'hr':
+            interval[0] = float(interval)[0] * 3600
+            interval[-1] = 's'
+        elif interval[-1] == 'min':
+            interval[0] = float(interval[0]) * 60
+            interval[-1] = 's'
+        elif interval[-1] == 'msec':
+            interval[0] = float(interval[0]) / 1000
+            interval[-1] = 's'
+        elif interval[-1] == 'µs':
+            interval[0] = float(interval[0]) / 1000000
+            interval[-1] = 's'
+    else:
+        interval[0] = float(interval[0])
+        interval[-1] = 's'
+    # get units and names
+    orig_units = []
+    orig_names = []
+    for index1 in range(3, len(header[-1]) + 8, 2):
```

Comment:
I think that @eurunuela 's point is still not met. Can you maybe comment this line to make it more clear (so we can help you with a name for the variable?)

Reply:
so basically the variable is an index than used into the header retreats for every channel the original name and units

Reply:
we could call it ``retriving_indx```?